conf/genomes.config

```diff
@@ -17,6 +17,8 @@ params {
             bwamem2             = "${params.local_genomes}/grch37_homo_sapiens_-d5-.fasta.{0123,amb,ann,bwt.2bit.64,pac}"
             call_interval       = ""
             gnomad              = "${params.local_genomes}/grch37_gnomad_reformated_-r2.1.1-.vcf.gz"
+            gnomad_af           = ""
```

Comment:
No strong reason really. Since the pipeline is in development stage now, I am mainly focusing on testing how the pipeline works both when I give it a file and when I don't. :)